tailwind.config.js

```diff
@@ -20,4 +20,11 @@ module.exports = {
     },
   },
   plugins: [],
+  mode: 'jit',
```

Comment:
Do we need just in time? Or was this just a test which is no longer necessary?

Reply:
it works without so Im removing it